server/container_create_linux.go

```diff
@@ -799,6 +800,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 		return nil, err
 	}
 
+	if sb.Annotations()[crioann.UmaskAnnotation] != "" {
+		umask, err := strconv.ParseUint(sb.Annotations()[crioann.UmaskAnnotation], 10, 32)
```

Comment:
what would you think about having this be a hex string instead of an int?  I'm sure a library exists for this, but I'm not sure where.

Reply:
@haircommander  yes. it's better if a library can be used. 
So you mean we can use a library that converts the hex string to uin32?

Reply:
so it seems podman uses a combination of [ParseUint](https://github.com/containers/podman/blob/afc84151fcc0c3ee329beca31e1c6a06d656a4a3/libpod/container_internal_common.go#L357-L362)

 and a [regex to validate](https://github.com/containers/podman/blob/3f76f29adb070b3481843d33f75d05b8c8aebdfa/libpod/options.go#L1782)

I can't find a library. I'm fine using that 

small correction of myself: it's not hex but octal 

Basically, I'd expect a user to be able to configure `022` and it work